notebooks/notebooks/paper/katago-new-ckpt-evals.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    plt.style.use(
</pre><p>is setting the style here necessary considering that we set it at the beginning of the notebook and that when you changed the style a few cells above, you used <code>with plt.style.context</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Good catch. Have removed.</p>